src/exo/platforms/x86.py

```diff
@@ -99,6 +99,15 @@ def mm256_add_ps(out: [f32][8] @ AVX2, x: [f32][8] @ AVX2, y: [f32][8] @ AVX2):
         out[i] = x[i] + y[i]
 
 
+@instr("{dst_data} = _mm256_add_ps ({src_data}, {dst_data});")
```

Comment:
We should keep the generated code consistently formatted.
```suggestion
@instr("{dst_data} = _mm256_add_ps({src_data}, {dst_data});")
```

Reply:
Done